### DISCUSSION TITLE
自定义prompt实现

### DISCUSSION THREAD

**Discussion by pwh-pwh** [pwh-pwh/aiwechat-vercel#7] [Ideas]
现在有个问题想调研一下，自定义prompt使用环境变量来配置还是使用指令配置(类似tg一样如发送 /prompt 你扮演一个老师 即可配置)

**Comment by tsfans** [comment]
建议使用指令配置，存到数据库里，方便后续拓展

**Comment by QAbot-zh** [comment]
支持指令配置，灵活度高，省得想用新prompt的时候还要修改环境变量重新部署。
理想状态是预存一些系统预设，然后也支持通过指令修改当前的预设，灵活度极大（目前我的微信机器人就是这样设计的）
